### DISCUSSION TITLE
v4.1

### DISCUSSION THREAD

**Discussion by RussKie** [gitextensions/gitextensions#10958] [General]
### Release Notes Highlights
Requires: **.NET 6.0 Desktop Runtime [v6.0.16](https://dotnet.microsoft.com/download/dotnet/thank-you/runtime-desktop-6.0.16-windows-x64-installer)** or later

* Git 2.40.1 is recommended
* Improvements in WSL2 handling, e.g., automatically convert `\\wsl.localhost\` to `\\wsl$\` paths
* Dashboard perf improvements
* Avatar perf improvements
* Blame perf improvements
* Reflog is decoupled from branch filtering and turned into an additive filter
* <kbd>Ctrl+Enter</kbd> creates commit when released and not pressed
* Windows Explorer integration can now be registered and unregistered from Settings

Full change log: https://github.com/gitextensions/gitextensions/blob/release/4.0/GitUI/Resources/ChangeLog.md#version-41-01-may-2023

![Downloads](https://img.shields.io/github/downloads/gitextensions/gitextensions/v4.1/total.svg)

### Known Issues:

* [ ] View Layout UI Bug
    ![Screenshot_1](https://github.com/gitextensions/gitextensions/assets/5248041/d30e94cf-2c7d-48a6-86fe-ad5fd12baa01)
    **Workaround:**
    Change info panel to appear on right, then back to bottom.
    ![image](https://github.com/gitextensions/gitextensions/assets/967010/22df3669-5a25-4381-a062-ce95d8c05f6a)

<hr /><em>This discussion was created from the release <a href='https://github.com/gitextensions/gitextensions/releases/tag/v4.1'>v4.1</a>.</em>

**Comment by sjh37** [comment]
I had the same issue
![image](https://github.com/gitextensions/gitextensions/assets/4190514/4f64c248-0511-4547-b773-bdcea9561836)

However, selecting this fixed it
![image](https://github.com/gitextensions/gitextensions/assets/4190514/42c88525-e5f8-4414-8e37-8974415e9a71)

**Comment by szdolhan-abs** [comment]
Still without dark theme?

**Reply by vbjay** [reply]
Instead of commenting here you could have read the dark theme issues and checked their status along with reading current and previous release notes.
